tsconfig.test.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true
```

Comment:
`ts-jest` set `module` to `commjs` unless set `allowSyntheticDefaultImports` to `true`.